app/helpers/markdown_helper.rb

```diff
@@ -4,13 +4,41 @@ module MarkdownHelper
   def camofy(markdown)
     return unless markdown
 
-    markdown.gsub(markdown_img_regex) do
+    markdown = sub_markdown(markdown)
+    sub_html(markdown)
+  end
+
+  def sub_markdown(text)
+    text.gsub(markdown_img_regex) do
       "![#{Regexp.last_match(1)}](#{camo(Regexp.last_match(2))}#{Regexp.last_match(3)})"
     end
   end
 
+  def sub_html(text)
+    text.gsub(html_img_regex) do
+      preceding_src = Regexp.last_match(1)
+      quote_mark = Regexp.last_match(2)
+      url = Regexp.last_match(3)
+      ending = Regexp.last_match(4)
+      "<img#{preceding_src} src=#{quote_mark}#{camo(url)}#{quote_mark}#{ending}"
+    end
+  end
```

Comment:
I had to split things up into separate methods because of a linter limit on method line count

Reply:
I think it's also more readable this way so I agree with the linter

Reply:
makes 2